resources/NetworkRestSocket.js

```diff
@@ -79,17 +79,30 @@ class NetworkRestSocket extends NetworkRest {
       return io.socket.isConnected();
    }
 
-   // Wrap the call with senrty span for perfromance tracking
-   async salSend(params) {
-      const shortRoute =
-         params.url.match(/https?:\/\/[^/]+(\/.+)/)?.[1] ?? params.url;
-      return Sentry.startSpan(
-         { name: shortRoute, op: "websocket.client" },
-         async () => await this._salSend(params)
-      );
-   }
+   salSend(params) {
+      let route, query;
+      try {
+         // Extract paramitized route (ex: `/app_builder/model/:ID`) for performance tracking
+         [, route, query] = params.url.match(
+            /https?:\/\/[^/]+(\/[^?]+)\??(.*)/
+         );
+         route = route.replace(/\b[a-fA-F\d-]{36}\b/g, ":ID");
+         performance.mark(route, {
+            op: "websocket.client",
+            data: {
+               http: {
+                  query: query || undefined,
+                  method: params.method,
+               },
+               url: params.url,
+            },
+         });
+      } catch (err) {
+         this.AB.notifyDeveloper(err, {
```

Comment:
shouldn't this be: `this.AB.notify.developer()`

Reply:
thanks